Test for the stats module

BasileiosKal opened the pull request:
<!-- Your title above should be a short description of what
was changed. Do not include the issue number in the title. -->

#### References to other Issues or PRs
<!-- If this pull request fixes an issue, write "Fixes #NNNN" in that exact
format, e.g. "Fixes #1234" (see
https://tinyurl.com/auto-closing for more information). Also, please
write a comment on that issue linking back to this pull request once it is
open. -->
Related to issue #16318

#### Brief description of what is fixed or changed
Added test to the stats module, for the "compute_expectation" function of the SingleContinuousDomain class (crv.py file) and for the "quantile" function of the NormalDistribution class (crv_types.py file).

#### Release Notes

<!-- Write the release notes for this release below. See
https://github.com/sympy/sympy/wiki/Writing-Release-Notes for more information
on how to write release notes. The bot will check your release notes
automatically to see if they are formatted correctly. -->

<!-- BEGIN RELEASE NOTES -->
NO ENTRY
<!-- END RELEASE NOTES -->

BasileiosKal committed:
Two tests for the stats module

Tests to increase the coverage of the stats module
Test 1. Test for the compute_expectation function of the
        SingleContinuousDomain class (crv.py file). Checks if
        a Value is raised when the variables are not equal.
Test 2. Test for the quantile function of the NormalDistribution
        (crv_types.py file). Checks if the function gives the
        correct result
Related to issue 16318.

BasileiosKal committed:
Update test_continuous_rv.py

smitgajjar commented on sympy/stats/tests/test_continuous_rv.py:
```diff
@@ -1497,3 +1497,15 @@ def test_ContinuousDistributionHandmade():
+
+def test_issue_16318():
+    #test compute_expectation function of the SingleContinuousDomain
+    x,y = symbols('x y')
```
Keep a space after comma and before y according to PEP8

BasileiosKal replied:
Thank you for your feedback! I removed the section as x, y are already defined above.

Smit-create commented on sympy/stats/tests/test_continuous_rv.py:
```diff
@@ -1497,3 +1497,15 @@ def test_ContinuousDistributionHandmade():
+    sigma = Symbol("sigma", positive=True)
+    p = Symbol("p")
+    X = Normal("x", mu, sigma)
+    assert mu + sqrt(2)*sigma*erfinv(2*p - 1) == quantile(X)(p)
```
This one is not required as it is already tested in `def test_multiple_normal():`. We don't need multiple tests asserting the same thing.

BasileiosKal replied:
Yes you're right! I didn't notice because the "quantile" function appears to not be covered on Codecov. I removed this section. Thank you very much for the help!

Smit-create commented on sympy/stats/tests/test_continuous_rv.py:
```diff
@@ -1497,3 +1497,15 @@ def test_ContinuousDistributionHandmade():
+
+def test_issue_16318():
+    #test compute_expectation function of the SingleContinuousDomain
+    x,y = symbols('x y')
```
No need to define this, as they are already defined at the top of the file.

BasileiosKal replied:
Done! Thank you.

BasileiosKal committed:
Test to increase the coverage of the stats module

Test for the compute_expectation function of the
SingleContinuousDomain class (crv.py file). Checks if
a Value is raised when the variables are not equal.
